opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -84,7 +85,41 @@ impl UserEventsExporter {
         ];
 
         for &level in levels.iter() {
-            eventheader_provider.register_set(level, keyword);
+            otel_debug!(
+                name: "UserEventsExporter.RegisterEvent",
+                level = level.as_int(),
+                keyword = keyword,
+            );
+            let event_set = eventheader_provider.register_set(level, keyword);
+            match event_set.errno() {
+                0 => {
+                    otel_debug!(name: "UserEventsExporter.RegisteredEvent",  event_set = format!("{:?}", event_set));
+                }
+                95 => {
+                    otel_debug!(name: "UserEventsExporter.TraceFSNotMounted", event_set = format!("{:?}", event_set));
+                }
+                13 => {
+                    otel_debug!(name: "UserEventsExporter.PermissionDenied", event_set = format!("{:?}", event_set));
```

Comment:
```suggestion
                    otel_debug!(name: "UserEvents.PermissionDenied", event_set = format!("{:?}", event_set));
```

Or something. UserEventExporter is already in the target, so no need to duplicate?

Reply:
Changed it to `UserEvents` at all the places